denis.roman/task1.rb

```diff
@@ -0,0 +1,6 @@
+Arr = Array.new(3) { Array.new(3) { rand -10...10 } }
```

Comment:
Please use local variables here, that's what we were talking about on the lection.